.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
@temanisparsh Let's add a more comprehensive .gitignore so we don't have to mess around with it later.
You can check out https://www.toptal.com/developers/gitignore